### PR TITLE
What am I doing right now as CI signal lead

### PR DESCRIPTION
**Pull request by spiffxp**
This could be worded / structured better but I want it to live someplace more "real" than [a PR comment](https://github.com/kubernetes/test-infra/pull/5306#issuecomment-341277823)

/cc @detiber 
you're secondary

/cc @enisoc 
you're the release lead, does this sound right

/cc @ericchiang 
you did this last time, what am I missing

### CODE REVIEW

File: release-process-documentation/release-team-guides/ci-signal-playbook.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+TODO: This could be worded / structured better but I want it to live someplace more "real" than [a PR comment](https://github.com/kubernetes/test-infra/pull/5306#issuecomment-341277823)
+
+## Status Quo
+
+Currently, here's what I'm doing as the CI Signal role for release 1.9:
+
+### Checking [sig-release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) daily
+
+- if a job is failing in one of the meta-stages (Up, Down, DumpClusterLogs, etc), I find the owning sig
```
Comment (ericchiang) [inline release-process-documentation/release-team-guides/ci-signal-playbook.md]: Just want to stress to open issues for _everything_. If you talk to someone on slack and they say they're fixing it, still open an issue and label it with `/priority failing-test` for visibility. Not doing this bit me a couple times last release.

File: release-process-documentation/release-team-guides/ci-signal-playbook.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- if a job is failing because a specific test case is failing, and that test case has a [sig-foo] in its name, I go bug sig-foo
+- with unit test case failures, I try to infer the sig from the path, or OWNERS files in it, otherwise I find the owning sig to help
+- with verify failures, I try to infer the failure from the log, otherwise I find the owning sig to help
+- if a test case is failing in one job consistently, but not others, both the job owner and test case owner are responsible for identifying why this combination is different
+
+### Pinging SIG's
+
+I do this by using @kubernetes/sig-foo-test-failures teams when I file issues, or going to the #sig-foo slack channel if I need routing help and know someone's around
+
+If a job as a whole is failing, I file an issue in kubernetes/kubernetes titled:`[job failure] job name issue`
```
Comment (ericchiang) [inline release-process-documentation/release-team-guides/ci-signal-playbook.md]: Are issues not included in the milestone?

Comment (spiffxp) [reply release-process-documentation/release-team-guides/ci-signal-playbook.md]: ... good point, they should be, and I don't currently have the power to add issues to the milestone

I have gotten by in interim by bugging someone else on the team with milestone powers, but if we can't get prow or something to add milestone in time, I should ask for the privs.

Comment (spiffxp) [reply release-process-documentation/release-team-guides/ci-signal-playbook.md]: Opened https://github.com/kubernetes/test-infra/issues/5344

**Comment by spiffxp** [comment]
>Was monitoring upgrade tests split into a different position again? Does this role no longer keep an eye https://k8s-testgrid.appspot.com/sig-release-master-upgrade ?

@ericchiang I hadn't been looking at upgrade tests as blockers for cutting alphas.  TBH I don't have the time to scrub both of these boards and all related issues daily at the moment.  Best I can offer is best-effort on the upgrade board unless I can get help from someone else.  

I'll start trying to at least glance at release-master-upgrade prior to code slush, just to get a feel for where those tests are at prior to creating all the release branch CI.

**Comment by spiffxp** [comment]
I explicitly mention filing milestone issues, and talk about release 1.y instead of 1.9.  I added a bit about requirements for the role up top.

I'm ready for this to merge as-is and iterate with later PR's.  Don't have write privs to this repo.

**Pull request merged**